src/main/java/org/jabref/gui/ai/components/summary/SummaryShowingComponent.java

```diff
@@ -32,20 +38,49 @@ public SummaryShowingComponent(Summary summary, Runnable regenerateCallback) {
 
     @FXML
     private void initialize() {
-        summaryTextArea.setText(summary.content());
+        initializeWebView();
+        updateContent(false); // Start in plain text mode
+        updateInfoText();
+    }
+
+    private void initializeWebView() {
+        contentWebView = WebViewStore.get();
+        contentWebView.setMinHeight(100);
+        contentWebView.setPrefHeight(300);
+
+        // Apply styles directly to the WebView
+        contentWebView.setStyle("-fx-font-family: Sans;");
```

Comment:
No, it should respect JabRef's current font. Yeah, I might've used the word "main", but by "main" I meant the font that is used in JabRef's runtime on user's machine.

By default, it might be Sans, but you should find a way of getting JabRef's current main font. So that if I change font family and/or size -- text in `WebView` will also change (except plain Markdown instead of rendered one, it can use monospaced font)

Reply:
Found the `ThemeManager`, injected it, and installed the css into `WebEngine`. Works good!